app/views/applications/new.html.erb

```diff
@@ -26,8 +26,13 @@
     <%= f.form_field :text_field, :email, 'Email' %>
     <%= f.form_field :text_field, :email_confirmation, 'Email Confirmation' %>
 
-    <% if @event.travel_funded? %>
-      <div class="form_field">
+    <div class="form_field">
```

Comment:
Hm, I think this also doesn't belong here anymore. We only have the visa checkbox now. 
```
<% if @event.travel_funded? %>
    <div class="form_field">
      <%= label_tag do %>
      <%= f.check_box :visa_needed %>
        I will need a visa to attend
      <% end %>
    </div>
  <% end %>
```
The problem with the checkbox is only on the terms and conditions, so I think you can leave it as it was before.